### PR TITLE
fix(CSSLayout): Remove unnecessary whole pixel rounding from the CSSL…

### PR DESCRIPTION
**Pull request by rozele**
…ayout algorithm.

### CODE REVIEW

File: ReactWindows/ReactNative/UIManager/ReactShadowNode.cs [hunk 1]
```diff
@@ -25,10 +25,10 @@ public class ReactShadowNode : CSSNode
         private bool _isLayoutOnly;
         private int _totalNativeChildren = 0;
         private ReactShadowNode _nativeParent;
-        private double _absoluteLeft;
-        private double _absoluteTop;
-        private double _absoluteRight;
-        private double _absoluteBottom;
+        //private double _absoluteLeft;
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/ReactShadowNode.cs]: delete.

File: ReactWindows/ReactNative/UIManager/OnLayoutEvent.cs [hunk 2]
```diff
@@ -6,12 +6,12 @@ namespace ReactNative.UIManager
 {
     class OnLayoutEvent : Event
     {
-        private int _x;
-        private int _y;
-        private int _width;
-        private int _height;
+        private double _x;
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/OnLayoutEvent.cs]: dims instead?

Comment (mpodwysocki) [inline ReactWindows/ReactNative/UIManager/OnLayoutEvent.cs]: @rozele do you mean a dims object instead of just those local variables?

Comment (mpodwysocki) [inline ReactWindows/ReactNative/UIManager/OnLayoutEvent.cs]: @rozele makes sense to use the Dimensions class here as it is extensible and not just a bunch of params

**Pull request merged**